learn/advanced/chronological_filtering_sorting.md

```diff
@@ -0,0 +1,109 @@
+# Filtering and sorting with chronological values
+
+In this guide, you will learn about Meilisearch's approach to date and time values, how to prepare your dataset for indexing, and how to chronologically sort and filter search results.
+
+## Preparing your documents
```

Comment:
I present the ideal case (dataset contains timestamps) first, then the likely scenario of a dataset using datetime.

Alternatively, we could separate these two cases in two sections, or even make the datetime scenario a subsection of `## Preparing your documents`. 

Any opinions?

Reply:
I don't get this 👀